src/index.ts

```diff
@@ -104,7 +104,7 @@ export class MidiOut {
   }
 
   /**
-   * Select a voice that's using a cents offset combatible channel or the oldest voice if nothing can be re-used.
+   * Select a voice that's using a cents offset compatible channel or the oldest voice if nothing can be re-used.
```

Comment:
I think you had more typo corrections in there. If you include them all, I won't mind the irrelevant id change.

Reply:
lines 137 and 140 "in" changed to "on"... should I commit and send pull request? just learning how to do it, sorry to be annoying!